### ISSUE TITLE
nameGap does not appear to work

### ISSUE THREAD

**Issue by guybrettrobertson** [JohnCoene/echarts4r#403]
The nameGap argument in `echarts4r::e_y_axis` does not appear to generate any space between the axis name and axis line, as it is supposed to. Perhaps I am misunderstanding something?
```
data.frame(a` = c(0, 1, 2, 3, 4, 5), b = c(0, 1, 2, 3, 4, 5)) |>
  echarts4r::e_chart(a) |>
  echarts4r::e_line(b) |>
  echarts4r::e_y_axis(
    name = 'Apples',
    nameLocation = list('center'),
    nameRotate = list(90),
    nameGap = list(40),
    nameTextStyle = list(align = 'center'))
```
![chart](https://user-images.githubusercontent.com/65654871/150132445-e44f9a9d-a6a6-4700-b13c-1c2bd3026714.png)

**Comment by JohnCoene** [comment]
I'm not behind my laptop but it looks like `nameGap` should not be an `array`, just an `integer`.

Does `nameGap = 50` work?

**Comment by guybrettrobertson** [comment]
Thanks for the suggestion but it doesn't change it.

As a side note, I'd also be interested to know when the 'additional arguments' should be entered as a list vs a single value. Is there a page that explains how to use the additional arguments?

**Comment by JohnCoene** [comment]
There is a small guide for those options [here](https://echarts4r.john-coene.com/articles/get_started.html#navigate-options-1)
For the `nameGap` it is passed along so I don't see any bug here.

```r
data.frame(a = c(0, 1, 2, 3, 4, 5), b = c(0, 1, 2, 3, 4, 5)) |> 
      echarts4r::e_chart(a) |> 
      echarts4r::e_line(b) |> 
      echarts4r::e_y_axis( 
        name = 'Apples', 
        nameLocation = 'center', 
        nameRotate = 90, 
        nameGap = 25, 
        nameTextStyle = list(align = 'center')) 
```

Doesn't that do what you want?

**Comment by guybrettrobertson** [comment]
Yes thank you! It looks like `nameLocation = list('center')` was also causing problems for me.

I will make sure to try removing the `list` call from now on.

**Comment by JohnCoene** [comment]
If you see in the original echarts.js documentation that these options expect `array`s (`['square', 'brackets']`) then use a `list("square", "brackets")`, otherwise no need for `list`.
